src/MensaWidget/MWMeal.class.st

```diff
@@ -0,0 +1,96 @@
+"
+This class is a structure for saving meal information about a meal from the OpenMensa-API.
+"
```

Comment:
we also need uniformity in the class comment! I personally like the default format which also includes the type of instance vars and a description for them.